mon-pix/package.json

```
@@ -28,7 +28,7 @@
     "lint:js": "eslint . --cache --cache-strategy content",
     "lint:js:fix": "npm run lint:js -- --fix",
     "lint:js:uncached": "eslint .",
-    "lint:scss": "stylelint app/styles/*.scss 'app/styles/**/*.scss'",
+    "lint:scss": "stylelint app/styles/*.scss 'app/styles/**/*.scss' 'app/pods/**/*.scss'",
```

C'est quoi le risque de mettre ça : 
```suggestion
    "lint:scss": "stylelint 'app/**/*.scss'",
```

---

Excellente question, je vais tester cet aprem

---

Tant qu'on entoure bien de quotes :D